.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ on:
   workflow_dispatch:
 
 concurrency:
-  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
```

Comment:
Will `github.event.pull_request.number` be defined when bors is running on those branches?

Reply:
No, it will just be empty. It's fine because workflows runs on bors branches are supposed to be run one at a time.

Reply:
I see. So if I understand correctly this makes try and r+ conflict with each other, I'm not sure we want that?

Reply:
Maybe we need to include the branch name instead of the sha?

Reply:
Multiple r+ are queued and ran one by one unless batched together under the same priority, so it doesn't matter.